app/configurator/configurator-state.spec.tsx

```diff
@@ -153,7 +153,11 @@ describe("initChartFromLocalStorage", () => {
   it("should initialize from localStorage if valid", async () => {
     localStorage.setItem(
       getLocalStorageKey("viz1234"),
-      JSON.stringify({ state: "CONFIGURING_CHART", ...fakeVizFixture })
+      JSON.stringify({
+        state: "CONFIGURING_CHART",
+        published_state: "PUBLISHED",
+        ...fakeVizFixture,
+      })
```

Comment:
Maybe it also would make sense to apply migration here, to forget about updating this when config schema changes?

Reply:
Will do in another PR.